pageserver/src/tenant/timeline.rs

```diff
@@ -3829,6 +3831,8 @@ impl Timeline {
                     .await;
 
                 if let Ok(size) = &result {
+                    info!("layer file downloaded finished");
+
```

Comment:
Cannot see why we need this in addition instead of just `successful` info and `failed` error?

Reply:
Because we can fail after we've successfully downloaded the layer file.

Reply:
Hm, the `cargo test` step takes the same amount of time as in earlier commits